index.html

```diff
@@ -84,6 +84,148 @@ <h5 class="text-purple skillboxPL">Skills</h5>
+                            dummy text.
+                        </div>
+                        <div class="langFrame-work">
+                            <div class="lang-box separator-lang" id="css">
+                                <h4 class="language-text">css</h4>
+                            </div>
+                            <div class="lang-box separator-lang" id="html">
+                                <h4 class="language-text">html</h4>
+                            </div>
+                            <div class="lang-box separator-lang" id="bootstrap">
+                                <h4 class="language-text">bootstrap</h4>
+                            </div>
+                            <div class="lang-box" id="ruby">
+                                <h4 class="language-text">Ruby</h4>
+                            </div>
+                        </div>
+                        <br>
+                        <br>
+                        <button class="projectButton-work pButtonW-text">
+                            See Project
+                        </button>
+                    </div>
+                    <div class="workCard-container">
+                        <img class="image-work" src="src/img/img_Placeholder.png" alt="Image Placeholder">
+                        <div class="title-work">Multi-Post Stories</div>
+                        <div class="content-work">
+                            A daily selection of privately 
+                            personalized reads; no accounts 
+                            or sign-ups required. has been the 
+                            industry's standard dummy text 
+                            ever since the 1500s, when an 
+                            unknown printer took a standard 
+                            dummy text.
+                        </div>
+                        <div class="langFrame-work">
+                            <div class="lang-box separator-lang" id="css">
+                                <h4 class="language-text">css</h4>
+                            </div>
+                            <div class="lang-box separator-lang" id="html">
+                                <h4 class="language-text">html</h4>
+                            </div>
+                            <div class="lang-box separator-lang" id="bootstrap">
+                                <h4 class="language-text">bootstrap</h4>
+                            </div>
+                            <div class="lang-box" id="ruby">
+                                <h4 class="language-text">Ruby</h4>
+                            </div>
+                        </div>
+                        <br>
+                        <br>
+                        <button class="projectButton-work pButtonW-text">
+                            See Project
+                        </button>
+                    </div>
+                </div>
+            </section>
         </main>
```

Comment:
- [ ] **[REQUIRED CHANGES]** For this milestone, you are supposed to do the `About myself` section as well, [as described in the activity](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md).

Don't worry tho! This section is extremely similar to the one that you just implemented. You can do it :+1: